packages/site/plugins/react-router/browser.api.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { useBasepath, useStaticInfo } from 'react-static'
+import { BrowserRouter, StaticRouter, withRouter } from 'react-router-dom'
+
+const Location = withRouter(({ children, location }) => children(location))
+
+export default ({ RouterProps: userRouterProps = {} }) => ({
+  Root: PreviousRoot => ({ children }) => {
+    const routerProps = { basename: useBasepath() || '' }
+    if (routerProps.basename) routerProps.basename = `/${routerProps.basename}`;
+    const staticInfo = useStaticInfo()
+
+    // Test for document to detect the node stage
+    let Router
```

Comment:
Semi-colons a little bit inconsistent?

Reply:
that's because it's a vendored file; I'll make it more consistent